src/modules/Dropdown/Dropdown.js

```diff
@@ -335,6 +340,22 @@ export default class Dropdown extends Component {
     }
   }
 
+  removeItemOnBackspace = (e) => {
+    debug('removeItemOnBackspace()')
+    debug(keyboardKey.getName(e))
+    if (keyboardKey.getCode(e) !== keyboardKey.Backspace) return
+
+    const { multiple, search } = this.props
+    const { searchQuery, value } = this.state
+
+    if (searchQuery || !search || !multiple || !value) return
+
+    e.preventDefault()
+
+    // remove most recent value
+    this.onChange(e, _.dropRight(value))
```

Comment:
In case this component is not controlled (has no value / onChange props) then we need to handle updating the value ourselves.  We'll need a `this.setValue()` call just before this to support both controlled and uncontrolled dropdowns:

```js
// remove most recent value
const newValue = _.dropRight(value)

this.setValue(newValue)
this.onChange(e, newValue)
```

If the component is not controlled, `setValue` will succeed in setting the new value to state.  If it is controlled, `setValue` attempt will be ignored.  In this case, broadcasting the change via onChange will change our props from the outside and we'll take that in as the new value.